### ISSUE TITLE
The option `case-sensitive = false` invalidates `use-id-as-short = true`

### ISSUE THREAD

**Issue by tplobo** [cgnieder/acro#275]
Hi there @cgnieder, thanks for the package, it is great!
I currently have a difficulty with the combination of the `case-sensitive` and `use-id-as-short` options.

I use the `case-sensitive = false` option to ensure acronyms will work in chapter headings, `\chaptermark` calls and the like. However, to my understanding, this currently invalidates the use of `use-id-as-short = true`, since using the setup below forces all short-versions of acronyms _in the text_ to assume lower-case format.

```
\acsetup{
    case-sensitive = false,     % Ignore case in IDs (needed for \chaptermark)
    use-id-as-short = true,     % Use ID as short version, unless specified
}

\DeclareAcronym{HDMI}{long = High Definition...}

\ac{HDMI}
```

prints
`High Definition... (hdmi)`
instead of
`High Definition... (HDMI)`

I understand that I could forcefully provide `\DeclareAcronym{HDMI}{short=HDMI, long = High Definition...}`, but this is what `use-id-as-short` should avoid.

---
**TLDR:** is there a way of ensuring that calls of `\ac`-like commands are case-insensitive when `case-sensitive = false`, but also print the short versions as defined when `use-id-as-short = true`? In the example above, I expect:

```
\DeclareAcronym{HdMi}{long = High Definition...}
\ac{hdmi}
```
to print 
`High Definition... (HdMi)`
and not
`High Definition... (hdmi)`
